Changes

```diff
@@ -17,6 +17,11 @@ Working version
 - #10902: Do not register empty code fragments in natdynlink.
   (David Allsopp, review by Xavier Leroy and Damien Doligez)
 
+- #10950: Do not use mmap to allocate Caml_state.
+  In order to reduce virtual memory usage, we statically allocate
```

Comment:
I think you're dynamically allocating the structure now..

Reply:
Right, good catch.